src/NativeShell/src/media/RemotePlayerService.java

```diff
@@ -175,20 +208,29 @@ private void notifyWithBitmap(Intent handledIntent, Bitmap largeIcon) {
             }
 
             mediaSession.setPlaybackState(stateBuilder.build());
-
-            PendingIntent pendingIntent = PendingIntent.getService(getApplicationContext(), 1, intent, 0);
             Notification.Builder builder = new Notification.Builder(this)
                     .setContentTitle(title)
                     .setContentText(artist)
+                    .setPriority(Notification.PRIORITY_LOW)
                     .setDeleteIntent(pendingIntent)
                     .setContentIntent(createContentIntent())
                     .setProgress(duration, position, duration == 0)
                     .setStyle(style);
 
-            builder.setOngoing(true);
+            // newer versions of android require notification channel to display
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                builder.setChannelId(channelId);
+                // color notification based on cover art
+                builder.setColorized(true);
+            }
+
+            // swipe to dismiss media
+            builder.setOngoing(false);
+            // dynamic media position
             builder.setShowWhen(true);
             builder.setUsesChronometer(true);
             builder.setWhen(System.currentTimeMillis() - position);
+            // privacy value for lock screen
             builder.setVisibility(Notification.VISIBILITY_PUBLIC);
```

Comment:
shouldn't this be configurable?

Reply:
It is a system setting, this option doesn't change visibility but just changes how the system setting interprets the notification. So setting it public means it's not important and if the user only wants to hide important notifications this isn't really one of them.

Reply:
What I meant is whether we should make a toggle to mark it as "publically unimportant", that is, whether user him/herself thinks if his/her media showing on otherwise protected lock screen is okay.

Reply:
That is still handled in the system setting, there are three options for granularity so I think this is somewhat low priority. Also, the app doesn't even have a native settings page yet so there wouldn't be a way to set this value. There are a few things I want to add settings for but this isn't really one of them.

Reply:
Well, let's wait for a request from some user then. I personally won't care about music names displayed on my lockscreen, so it's fine by me :)